### ISSUE TITLE
wthit crashes game loading

### ISSUE THREAD

**Issue by SirPeteThe1st** [badasintended/wthit#156]
### Describe the issue

After updating my Forge and the mods that I've downloaded for it, the WTHIT mod fails the loading process. Running Minecraft 1.18.2 on Forge version 40.1.73 on a late 2014 iMac with macOS Big Sur 11.6.8. I've created the necessary reports as instructed. If there is any more info you need, please contact me. I will be very available to help as I have been rendered unemployable and hopeless. 

### Log output and crash report

https://gist.github.com/SirPeteThe1st/a49b647cfd12d447eb9d27078b37c498

**Comment by deirn** [comment]
First, try without WTHIT and Bad Packets and see if it still crashes, if so then it's not my mod's fault. Second, try without Morph Mod (bmorph), it looks like its mixin failed to apply.

**Comment by SirPeteThe1st** [comment]
Good Day,

I decided to recreate the issue by doing a clean install of the most
updated Forge version with 1.18.2 and installing both WTHIT/Bad Packets and
Budschie's Morph Mod along with JEI, Journeymap, and a mod I made with
MCreator that plays the Wilhelm Scream when you die. Surprisingly, all of
the mods worked perfectly. There must have been some other kind of issue
that caused this crash.

Thank you for your swift response.

-PF
